src/stories/Page.stories.tsx

```diff
@@ -34,164 +28,104 @@ const argObject = {
+      <Block title="Open Source" imagePath="https://avatars.githubusercontent.com/u/9919?s=200&v=4" size={2} isEditable={true} actions={<Action.Link url="https://github.com"/>}>
+        <Block.Pills pills={["react.js", "vue.js", "angular.js", "ember.js", "svelte.js"]} />
+        <Block.List>
+          {openSource.map((project, index) => (
+            <Block.List.Item
               type={project.type}
               title={project.title}
               additional={project.additional}
               subtitle={project.subtitle}
               count={project.count && { value: project.count?.value, icon: project.count?.icon }}
+              key={`Block-${index}`}
             />
           ))}
-        </List>
-      </Section>
-      <Section>
-        <Header title="Web Projects" icon="GlobeAltIcon" />
-        <List>
-          {projects.map((project) => (
-            <List.Item
+        </Block.List>
+      </Block>
+      <Block imagePath="https://avatars.githubusercontent.com/u/24602613?s=200&v=4" size={2} isEditable={true} title="Deployed Projects">
+        <Block.List>
+          {projects.map((project, index) => (
+            <Block.List.Item
               type={project.type}
               title={project.title}
               additional={project.additional}
               subtitle={project.subtitle}
               image={project.image}
+              key={`Block-${index}`}
             />
           ))}
-        </List>
-      </Section>
-      <Section>
-        <Header
-          title="My Music"
-          icon="MusicalNoteIcon"
-          actions={
-            <Actions>
-              <Action.Link url="https://github.com" text="Spotify profile" />
-            </Actions>
-          }
+        </Block.List>
+      </Block>
+      <Block title="My Music" imagePath="https://avatars.githubusercontent.com/u/251374?s=200&v=4" size={2} isEditable={true} actions={<Action.Link url="https://spotify.com"/>}>
+        <Block.Card
+          type="horizontal"
+          title="This album title"
+          subtitle="Artist Name"
+          image="https://placeimg.com/640/480/arch"
+          icon="PlayIcon"
         />
-        <Cards>
-          <Cards.Item
-            type="horizontal"
-            title="This album title"
-            subtitle="Artist Name"
-            image="https://placeimg.com/640/480/arch"
-            icon="PlayIcon"
-          />
-        </Cards>
-        <List>
+        <Block.List>
           {spotify.map((album, index) => (
-            <List.Item
+            <Block.List.Item
               index={index + 1}
               type={album.type}
               title={album.title}
               subtitle={album.subtitle}
               image={album.image}
               count={album.count && { value: album.count.value, icon: album.count.icon }}
+              key={`Block-${index}`}
             />
           ))}
-        </List>
-      </Section>
+        </Block.List>
+      </Block>
     </>,
   ],
   aside: [
```

Comment:
@NilsJacobsen Are we getting rid of aside and main or will it stay? Because we talked about a 3 column layout some days ago, whats your opinion on that?

Reply:
I don't want to integrate the grid library here, I feel like that's a lot of studio logic in the ui lib. So I thought maybe it's fine to just use a custom flex layout that has a similar outcome than the react-grid-lib. 
https://github.com/react-grid-layout/react-grid-layout